swagger/v0/swagger.yaml

```diff
@@ -1112,40 +1184,14 @@ paths:
                           - county
                           - postcode
                           - latLong
+                          - onsDistrictCode
+                          - localAuthority
                           additionalProperties: false
-                        membershipNumber:
-                          type: string
-                        name:
-                          type: string
-                        serialNumber:
-                          type: string
-                        email:
-                          type: string
-                        id:
```

Comment:
It looks like some of the properies have been removed from the `/api/v0/json/vacancy/list` endpoint. Is that intentional?

Reply:
nope, this was a mistake